src/dir.cr

```diff
@@ -64,7 +64,15 @@ class Dir
     EntryIterator.new(self)
   end
 
-  # Returns an array containing all of the filenames in the given directory.
+  # Returns an `Array(String)` containing all of the filenames in the given directory including "." and "..".
```

Comment:
This makes it sound like it will only return files, when it will also return subdirectories.  Not sure how to word it but perhaps something like:

```suggestion
  # Returns an `Array(String)` containing all of the filenames and subdirectory names in the given directory including "." and "..".
```

Reply:
Maybe `all the entries`?

Reply:
Specifying the type is redundant here (it's already specified as the return type definition).

```suggestion
  # Returns an array containing all of the filenames in the given directory including "." and "..".
```